### ISSUE TITLE
Problem in cleaning a text containing specific link

### ISSUE THREAD

**Issue by ahangarha** [s/preprocessor#29]
The following code cannot clean the text:

```python
import preprocessor as tp

text = "check this link: https://fa.wikipedia.org/wiki/%D8%AD%D9%85%D9%84%D9%87_%D8%A8%D9%87_%DA%A9%D9%88%DB%8C_%D8%AF%D8%A7%D9%86%D8%B4%DA%AF%D8%A7%D9%87_%D8%AA%D9%87%D8%B1%D8%A7%D9%86_(%DB%B1%DB%B8%E2%80%93%DB%B2%DB%B3_%D8%AA%DB%8C%D8%B1_%DB%B1%DB%B3%DB%B7%DB%B8) …"

print(tp.clean(text))
```

I think it gets stuck in some loop

**Comment by PandaWhoCodes** [comment]
Try this
`python -m pip install git+https://github.com/s/preprocessor`

**Comment by s** [comment]
Seems to be fixed in `v0.6.0`
